app/views/courses/preview/financial_support/_bursary.html.erb

```diff
@@ -16,11 +16,10 @@
   </p>
 
   <p class="govuk-body">
-    You may also be eligible for a <%= link_to "loan while you study","https://getintoteaching.education.gov.uk/funding-my-teacher-training/tuition-fee-and-maintenance-loans", class:"govuk-link" %>.
+    You may also be eligible for a <%= link_to "loan while you study", "https://getintoteaching.education.gov.uk/funding-my-teacher-training/tuition-fee-and-maintenance-loans", class:"govuk-link" %> - note that you'll have to apply for <%= link_to "undergraduate student finance", "https://www.gov.uk/student-finance", class:"govuk-link" %>.
```

Comment:
Can we fix the spacing? `class:"govuk-link"` -> `class: "govuk-link"` and in the other files too.

Reply:
Done, I've tidied up some of the other links on the same pages as well - is it worth getting a linter set up to catch this sort of thing?

Reply:
Cool 👍 We do run linting on CI but it seems to be missing these.